tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/OrganisationControllerTests.cs

```diff
@@ -410,5 +410,134 @@ public static async Task GetServiceRecipientsAsync_OrganisationExists_ReturnsThe
+                },
+            };
+
+            result.Should().BeEquivalentTo(new ActionResult<List<RelatedOrganisationModel>>(expected));
+        }
+
+        [Test]
+        public static async Task GetRelatedOrganisationsAsync_OrganisationExists_NoRelatedOrganisations_ReturnsEmptyList()
+        {
+            var organisation = OrganisationBuilder.Create(1).WithAddress(Address1).Build();
+
+            var controller = OrganisationControllerBuilder
+                .Create()
+                .WithGetOrganisationWithRelatedOrganisations(organisation)
+                .Build();
+
+            var result = await controller.GetRelatedOrganisationsAsync(organisation.OrganisationId);
+
+            result.Should().BeEquivalentTo(new ActionResult<List<RelatedOrganisationModel>>(new List<RelatedOrganisationModel>()));
+        }
+
+        [Test]
+        public static async Task GetRelatedOrganisationsAsync_OrganisationDoesNotExist_ReturnsNotFound()
+        {
+            var nonExistentOrganisationId = Guid.NewGuid();
+
+            var controller = OrganisationControllerBuilder
+                .Create()
+                .Build();
+
+            var result = await controller.GetRelatedOrganisationsAsync(nonExistentOrganisationId);
+
+            result.Should().BeEquivalentTo(new ActionResult<List<RelatedOrganisationModel>>(new NotFoundResult()));
+        }
+
+        [Test]
+        public static async Task CreateRelatedOrganisationAsync_RelatedOrganisationReturnsSuccess_ReturnsRelatedOrganisation()
+        {
+            var organisation = OrganisationBuilder.Create(1).Build();
+
+            var relatedOrganisation = OrganisationBuilder.Create(2).Build();
+
+            var relatedOrganisationModel = new CreateRelatedOrganisationModel() { RelatedOrganisationId = relatedOrganisation.OrganisationId };
+
+            var controller = OrganisationControllerBuilder
+                .Create()
+                .WithGetByIdWithRelatedAndGetByIdForRelatedAndUpdateAsync(organisation, relatedOrganisation)
+                .Build();
+
+            var result = await controller.CreateRelatedOrganisationAsync(organisation.OrganisationId, relatedOrganisationModel);
+
+            result.Should().BeOfType<NoContentResult>();
+        }
+
+        [Test]
+        public static async Task CreateRelatedOrganisationsAsync_OrganisationDoesNotExists_ReturnsNotFound()
+        {
+            var organisationId = Guid.NewGuid();
+
+            var controller = OrganisationControllerBuilder
+                .Create()
+                .Build();
+
+            var result = await controller.CreateRelatedOrganisationAsync(organisationId, new CreateRelatedOrganisationModel());
+
+            result.Should().BeOfType<NotFoundResult>();
+        }
+
+        [Test]
+        public static async Task CreateRelatedOrganisationsAsync_OrganisationExists_RelatedOrganisationDoesntExist_ReturnsBadRequest()
```

Comment:
Think it would be better if we spell out `DoesNot` here rather than use the contracted form.